Dragonflight/AlgetharAcademy/EchoOfDoragosa.lua

```diff
@@ -71,10 +78,14 @@ function mod:AstralBreath(args)
 	self:CDBar(args.spellId, 32.8)
 end
 
-function mod:PowerVacuum(args)
-	self:Message(args.spellId, "red")
-	self:PlaySound(args.spellId, "alarm")
-	self:CDBar(args.spellId, 23.1)
+function mod:UNIT_SPELLCAST_SUCCEEDED(_, _, _, spellId)
+	if spellId == 388820 then -- Power Vacuum
+		powervacuumCount = powervacuumCount + 1
+		self:StopBar(388822)
```

Comment:
There's no need for this StopBar because you're always restarting the bar on line 87 which would overwrite any previous bar with the same text.